cnf-certification-test/suite_test.go

```diff
@@ -264,3 +289,102 @@ func incorporateVersions(claimData *claim.Claim) {
 		ClaimFormat:  ClaimFormatVersion,
 	}
 }
+func StartServer() {
+	server := &http.Server{
+		Addr:         ":8084",           // Server address
+		ReadTimeout:  10 * time.Second,  // Maximum duration for reading the entire request
+		WriteTimeout: 10 * time.Second,  // Maximum duration for writing the entire response
+		IdleTimeout:  120 * time.Second, // Maximum idle duration before closing the connection
+	}
+	HandlereqFunc()
+
+	http.HandleFunc("/runFunction", RunHandler)
+
+	fmt.Println("Server is running on :8084...")
+	if err := server.ListenAndServe(); err != nil {
+		panic(err)
+	}
+}
+
+// Define an HTTP handler that triggers Ginkgo tests
+func RunHandler(w http.ResponseWriter, r *http.Request) {
+	Buf = bytes.NewBufferString(aString)
+	log.SetOutput(Buf)
+	logl.SetOutput(Buf)
```

Comment:
leftover?

Reply:
meant for the regular log

Reply:
the "regular" log seems to be a leftover, we should be using the same log everywhere (today, logrus)